Report.md

```diff
@@ -72,10 +66,17 @@ Stakeholders refer to individuals/entities that have an interest in the developm
 
 Design Thinking Approach
 ========================
-*3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*
-
 ## **Brainstormed Ideas and Reflection**
-*TODO*
+The following features were concluded from the brainstorming session. 
+Feature | Justification
+--- | ---
+Enhance Interactive Voice Response unit | Gain initial understanding of customer needs without the need for an in-person RM answering the call. This, in addition to the addition of hold music, were the best ideas ideated to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.
+Improve RM questionnaire  | Allows better initial classification of the RM by providing more parameters for CMC to score by.
+Added internal wiki of further package information | This was decided to be the best way to have all RMs confident with all packages, so they are more likely to make a sale to customers even if they lack experience with the destination.
+Feedback took implemented  |  Customer feedback tool was added so that the CMC can implement RM score in real-time after the call is undertaken. 
```

Comment:
Typos
```suggestion
Feedback tool |  Customer feedback tool was added so that the CMC could implement RM score in real-time after the call is undertaken. 
```